public/app/plugins/panel/timeline/utils.ts

```diff
@@ -62,7 +65,7 @@ export const preparePlotConfigBuilder: PrepConfig = ({
   colWidth,
   showValue,
 }) => {
-  const builder = new UPlotConfigBuilder(timeZone);
+  const builder = new UPlotConfigBuilder(timeZone as any);
```

Comment:
Can we avoid `as any` for such a well-known type? :)

Reply:
this will hopefully go away once we remove all mandatory time-isms from UPlotConfigBuilder

Reply:
the "as any" was not needed -- i think left over from early work with no typing effort